api/controllers/shabads.js

```diff
@@ -261,17 +261,21 @@ exports.search = async (req, res) => {
 exports.resultsInfo = async (req, res) => {
   const { VerseID } = req.params;
   const results = 20;
+  const verseArray = VerseID.split(',');
```

Comment:
On inputting non-numbers in verseID, it keeps on loading.

Reply:
This should be fixed now.